lib/node/fullnode.js

```diff
@@ -183,6 +185,17 @@ class FullNode extends Node {
       });
     }
 
+    if (this.config.bool('index-filters') || this.config.bool('bip157')) {
+      this.filterindex = new FilterIndexer({
+        network: this.network,
+        logger: this.logger,
+        blocks: this.blocks,
+        chain: this.chain,
+        memory: this.config.bool('memory'),
+        prefix: this.config.filter('index').str('prefix') || this.config.prefix
```

Comment:
The other indexers use `this.config.str('index-prefix', this.config.prefix)` for this.